recipe/meta.yaml

```diff
@@ -1,41 +1,47 @@
-{% set version = "1.1.7" %}
+{% set version = "2.0" %}
 
 package:
   name: pygmo
   version: {{ version }}
 
 source:
-  fn: pagmo-{{ version }}.tar.gz
-  url: https://github.com/esa/pagmo/archive/{{ version }}.tar.gz
-  sha256: 6d8fab89ef9d5d5f30f148225bf9b84b2e5a38997f3d68b85547840e9fd95172
+  url: https://github.com/esa/pagmo2/archive/v{{ version }}.tar.gz
+  sha256: 022af6bad2f1d083c2cacf4b121722541df80b6f168519a8eba0709cef02163a
 
 build:
-  number: 2
-
-  # https://github.com/esa/pagmo/issues/94
-  skip: true  # [win]
+  number: 0
+  skip: true  # [(win and py27) or win32]
 
 requirements:
   build:
     - toolchain
-    - python
     - cmake
-    - boost 1.63.*
+    - eigen
+    - nlopt
+    - boost     1.63.*
+    - numpy     x.x      # [not osx]
+    - numpy     1.12.*   # [osx]
```

Comment:
Yes, we have a matrix on the feedstocks. So timeouts shouldn't be an issue. Though a change like this will need a [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) to update the CIs' matrices.